packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.test.tsx

```diff
@@ -185,3 +186,22 @@ describe('Render calling with respective to props', () => {
     renderMock.mockRestore();
   });
 });
+
+describe('VerticalBarChart - mouse events', () => {
+  beforeEach(sharedBeforeEach);
+  afterEach(sharedAfterEach);
+
+  it('Should render callout on hover', async () => {
+    wrapper = mount(
+      <VerticalBarChart data={chartPoints} calloutProps={{ doNotLayer: true }} enabledLegendsWrapLines />,
+    );
+
+    // Wait for the chart to be resized
```

Comment:
why is this wait needed

Reply:
Without wait, the bars are not rendered because the barHeight stays 0